src/main/java/com/epam/ball/repository/comparators/BallZCoordinateComparator.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.ball.repository.comparators;
+
+import com.epam.ball.BallIdentifiable;
+import com.epam.ball.entity.Ball;
+import com.epam.ball.entity.Point3D;
+
+import java.util.Comparator;
+
+public class BallZCoordinateComparator implements Comparator<BallIdentifiable> {
+
+    @Override
+    public int compare(BallIdentifiable first, BallIdentifiable second) {
+        Point3D firstBallCenter = first.getCenter();
+        Point3D secondBallCenter = second.getCenter();
+
+        double firstZCenterCoordinate = firstBallCenter.getZ();
+        double secondZCenterCoordinate = secondBallCenter.getZ();
```

Comment:
for get method you can write in one line, don't create var if use it in one place